docs/source/data/errors.mdx

```diff
@@ -522,19 +541,14 @@ const setHttpPlugin = {
         if (response?.errors?.[0]?.message === 'teapot') {
```

Comment:
(previous line) you need to set all headers using lowercase header names now (it will throw otherwise)

(this line) there's an extra `response?.result?.errors` thing here (the structure will be different for defer/stream responses, thus the new "result" bit; also, having a sub-object that's exactly "the thing that goes in the body of the response" seems useful).

Reply:
Sweet, thank you!